base/uk.ac.stfc.isis.ibex.ui.scriptgenerator/src/uk/ac/stfc/isis/ibex/ui/scriptgenerator/views/ActionsViewTable.java

```diff
@@ -285,7 +304,8 @@ private boolean actionChanged(ScriptGeneratorAction tableAction, ScriptGenerator
 		return tableAction == null 
 				|| !tableAction.equals(newAction) 
 				|| tableAction.isValid() != newAction.isValid() 
-				|| tableAction.getEstimatedTime() != newAction.getEstimatedTime();
+				|| tableAction.getEstimatedTime() != newAction.getEstimatedTime()
+				|| tableAction.getEstimatedCustom() != newAction.getEstimatedCustom();
```

Comment:
We need to use `.equals()` to compare values here and also for getEstimatedTime above - otherwise we are checking _reference_ equality rather than _value_ equality.

(For `isValid`, `!=` is correct because `isValid` returns a primitive type - `boolean`). 

Note that I'd actually suggest using `Objects.equals()` rather than just `.equals` as it's null-tolerant.